README.md

```diff
@@ -8,15 +8,28 @@
 
 </div>
 
-This page is available as an easy-to-read website at [![https://ebookfoundation.github.io/free-programming-books/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=https%3A%2F%2Febookfoundation.github.io&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books%2F)](https://ebookfoundation.github.io/free-programming-books/)
+This page is available as an easy-to-read website at [![https://ebookfoundation.github.io/free-programming-books/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=https%3A%2F%2Febookfoundation.github.io&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books%2F)](https://ebookfoundation.github.io/free-programming-books/). Also try our new dynamic version of the website at [![https://ebookfoundation.github.io/free-programming-books-search/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=https%3A%2F%2Febookfoundation.github.io/free-programming-books-search&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books-search%2F)](https://ebookfoundation.github.io/free-programming-books-search/). 
```

Comment:
Rendering looks cramped .  Maybe start a new paragraph? (not specific to this PR, but I wonder whether it's obvious to people that the button is a link? Try to word the text so it won't grow stale in a month or two; I think we want to eventually remove or de-emphasize the static site. Also not specific to this PR, but with two versions of the site, maybe we can change the anchor text to "static site" and "Dynamic site"? in stead of an impenetrable url?

Reply:
Keeping it simple like Eric says...

```suggestion
This page is available as an easy-to-read websites. Access them clicking on [![https://ebookfoundation.github.io/free-programming-books/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=Static%20site&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books%2F)](https://ebookfoundation.github.io/free-programming-books/) or [![https://ebookfoundation.github.io/free-programming-books-search/](https://img.shields.io/website?style=flat&logo=www&logoColor=whitesmoke&label=Dynamic%20search%20site&down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books-search%2F)](https://ebookfoundation.github.io/free-programming-books-search/) links. 
```

Reply:
By writing the links users might not understand, wouldn't it be better to replace them with simple words?

Reply:
```suggestion
This page is available as an [easy-to-read website](https://ebookfoundation.github.io/free-programming-books/). Also try our new [dynamic version of the website](https://ebookfoundation.github.io/free-programming-books-search/). 
```

Reply:
Also, `dynamic version of the website` is a little vague, I'd include the fact that it has a working search tool.

Reply:
what if we used the official 'free-programming-books-search' github pages instead?

Reply:
> what if instead we use 'free-programming-books-search' as the (only) official github page?

I prefer both meanwhile parser works well at 99%. At this time search doesn't detect some resources. E.g.: spanish midudev

Reply:
I like @davorpa 's version

Reply:
I'll commit the version suggested by @davorpa